src/main/java/mediathek/config/MVConfig.java

```diff
@@ -22,6 +22,7 @@
 import com.jidesoft.utils.SystemInfo;
 import mSearch.Const;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'mSearch.Const'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)